PSReadLine/ConsoleLib.cs

```diff
@@ -13,10 +13,10 @@ internal class VirtualTerminal : IConsole
         // custom ReadKey implementation. This is not a public API, but it is
         // part of a private contract with that project.
         #pragma warning disable CS0649
-        private static Func<bool, ConsoleKeyInfo> _readKeyOverride;
+        private static readonly Func<bool, ConsoleKeyInfo> _readKeyOverride;
         #pragma warning restore CS0649
 
-        private static Lazy<Func<bool, ConsoleKeyInfo>> _readKeyMethod = new Lazy<Func<bool, ConsoleKeyInfo>>(
+        private static readonly Lazy<Func<bool, ConsoleKeyInfo>> _readKeyMethod = new Lazy<Func<bool, ConsoleKeyInfo>>(
             () => _readKeyOverride == null ? Console.ReadKey : _readKeyOverride);
```

Comment:
These 2 cannot have `readonly` added as they are actually a private contract between PSReadLine and PowerShellEditorService. They will be assigned via reflection in PSES.